core/arch/arm/plat-imx/sub.mk

```diff
@@ -1,7 +1,7 @@
 global-incdirs-y += .
 srcs-y += main.c
 
-srcs-$(CFG_MX6)$(CFG_MX7) += mmdc.c imx-common.c
+srcs-$(CFG_MX6)$(CFG_MX7) += imx_src.c  mmdc.c imx-common.c
```

Comment:
minor: 2 spaces

Reply:
Addressed in 03c1e4790198c7f0f064b28447ef8ce0e637376e